Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: docker-down
```

Comment:
Is this meant to be in here?

Reply:
Made Tech's (admittedly emerging) [Makefile standards](https://github.com/madetech/rfcs/blob/master/rfc-012-makefile-standards.md) recommended it.

These stop folders of this name getting confused with command by `make`. Often ran into it with `make test`, and though it's unlikely for these other folder names, I tend to just chuck it on 'em all. 

I _think_ you can do a one-liner to cover all the PHONYs, which might help if this hurts the readability, but I don't think this file is long enough to worry too much.

Reply:
Ahh, I see. I thought it was meant to be in the other PR, not in this one.

Reply:
Oh, right, yeah! This is branched off the Makefile PR, it won't show up in the final version of this'un.